les/downloader/downloader.go

```diff
@@ -229,7 +229,7 @@ func New(checkpoint uint64, stateDb ethdb.Database, stateBloom *trie.SyncBloom,
 		headerProcCh:   make(chan []*types.Header, 1),
 		quitCh:         make(chan struct{}),
 		stateCh:        make(chan dataPack),
-		SnapSyncer:     snap.NewSyncer(stateDb, nil),
+		SnapSyncer:     snap.NewSyncer(stateDb, ""),
```

Comment:
hm ```SnapSyncer:     snap.NewSyncer(stateDb, rawdb.HashScheme),``` As the previous PR, it points to HashScheme ?

Reply:
Do we miss update `les/downloader/statesync.go`

Reply:
As I checked, it was previously nil, and we will not use les anw, so we might skip this?

Reply:
Oke then let keep it like it, will comeback in case any issues happens.